packages/core/src/index.js

```diff
@@ -4,6 +4,8 @@ export { default as Badge } from './Badge'
 export { default as Banner } from './Banner'
 export { default as BlockLink } from './BlockLink'
 export { default as Box } from './Box'
+export { default as Breadcrumb } from './Breadcrumb'
```

Comment:
I wonder if exposing `Breadcrumb` might encourage people to go around the official `Breadcrumbs` component and try to roll their own if they have specific design needs, which probably isn't desirable.

Reply:
That's a good point